.github/workflows/codespell.yml

```
@@ -0,0 +1,23 @@
+name: Codespell
+
+on: ['pull_request']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - uses: actions/checkout@v3
+      
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v26.1
+        with:
+          files_ignore: pages.*/*/* # Currently ignores all other languages except English
+        
+      - uses: codespell-project/actions-codespell@master
+        with:
+          ignore_words_file: .github/codespell-ignorelist
+          only_warn: 1 # Will exit with 0 regardless
+          path: ${{ steps.changed-files.outputs.all_changed_files }} # Checks files in PR
```

The files should end with and empty line
```suggestion
          path: ${{ steps.changed-files.outputs.all_changed_files }} # Checks files in PR

```